common/src/main/java/de/michelinside/glucodatahandler/common/receiver/NamedBroadcastReceiver.kt

```diff
@@ -5,8 +5,7 @@ import android.content.Context
 import android.content.Intent
 import de.michelinside.glucodatahandler.common.GlucoDataService
 
-abstract class ReceiverBase: BroadcastReceiver() {
-    abstract fun getName(): String
+abstract class NamedBroadcastReceiver: BroadcastReceiver(), NamedReceiver {
```

Comment:
Renaming ReceiverBase will cause more changes than needed and may cause conflicts. 
I think, this should be done during refactoring, not for a new feature...
But it is ok for me, as there are not so many related changes...

Reply:
Just to confirm, do you want me to change it back?

Reply:
No, it is ok.